examples/extending_query/temporal_range.py

```diff
@@ -22,9 +23,8 @@
 class HasTemporal:
     """Mixin that identifies a class as having a timestamp column"""
 
-    timestamp = Column(
-        DateTime, default=datetime.datetime.utcnow, nullable=False
-    )
+    utc = partial(datetime.datetime.now, datetime.timezone.utc)
```

Comment:
I would avoid assigning this function inside the class

Reply:
Done